eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -6,11 +6,12 @@ parameters:
   Recursive: $false
   CheckLinkGuidance: $true
   Urls: '(Get-ChildItem -Path ./ -Recurse -Include *.md)'
-  BranchReplaceRegex: "^(${env:SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI}.*/(?:blob|tree)/)master(/.*)$"
+  BranchReplaceRegex: "(^${env:SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI}.*/(?:blob|tree)/)$(DefaultBranch)(/.*)$"
   BranchReplacementName: "${env:SYSTEM_PULLREQUEST_SOURCECOMMITID}"
   Condition: succeeded() # If you want to run on failure for the link checker, set it to `Condition: succeededOrFailed()`.
 
 steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
```

Comment:
Looks like this template is missing from the PR.

Reply:
Forgot to git add the new file